### PR TITLE
Unit Tests for NullBlamer (JENKINS-56719)

### PR DESCRIPTION
**Pull request by DaFloWaDa**
Adds tests for NullBlamer.

### CODE REVIEW

**Comment by DaFloWaDa** [comment]
Apparently adding the 'Work in Progress' label is not possible.
But actually there is not much more to do :man_shrugging:

File: src/test/java/io/jenkins/plugins/analysis/core/scm/NullBlamerTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+/**
+ * Tests the class {@link NullBlamer}
+ *
+ * @author Florian Hageneder
+ */
+class NullBlamerTest {
+
+    @Test
+    void shouldLogSkippedBlaming() {
+        Report report = mock(Report.class);
```
Comment (uhafner) [inline src/test/java/io/jenkins/plugins/analysis/core/scm/NullBlamerTest.java]: Here it makes sense to use the real class `new Report()`.

File: src/test/java/io/jenkins/plugins/analysis/core/scm/NullBlamerTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+        final NullBlamer blamer = new NullBlamer();
+        blamer.blame(report);
+
+        verify(report).logInfo(anyString(), any());
```
Comment (uhafner) [inline src/test/java/io/jenkins/plugins/analysis/core/scm/NullBlamerTest.java]: Then you can check for the actual value. (`assertThat(report.getInfoMessages()).contains(...)`). You can create a constant for the log message in `NullBlamer` and reuse it in the test.

File: src/test/java/io/jenkins/plugins/analysis/core/scm/NullBlamerTest.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ *
+ * @author Florian Hageneder
+ */
+class NullBlamerTest {
+
+    @Test
+    void shouldLogSkippedBlaming() {
+        Report report = mock(Report.class);
+
+        final NullBlamer blamer = new NullBlamer();
```
Comment (uhafner) [inline src/test/java/io/jenkins/plugins/analysis/core/scm/NullBlamerTest.java]: No `final` for local variables, see [coding style](https://github.com/uhafner/codingstyle).

**Comment by Vroni22** [comment]
Looks fine to me.

**Comment by uhafner** [comment]
Merged.

**Pull request closed without merging**